students/Chris_Kenyon/Lesson_6/Test_Mailroom.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+# Lesson_6 Activity 1 Test Mailroom Part 4
+
+from Mailroom_Pt4 import *
```

Comment:
it is preferred to use explicit imports rather than "star" imports, also python modules should be lower case -> `mailroom_pt4`